app/assets/javascripts/fortitude/components/navigationbar.js

```diff
@@ -4,16 +4,27 @@ title: Navigationbar
 name: js-navigationbar
 category: Javascript
 ---
-The `navigationbar` object is used for the main navigation of a website.
+The `navigationbar` object is used for the main navigation of your site or app. The javascript component allows you to show and hide the collapsed mobile navigationbar.
 
-<div class="note note--info">
-  <p class="styleguide">Navigationbars don't need JavaScript by default, but if you want the navigationbar to trigger the shade component then you'll want to use the JS Component</p>
-  <p class="styleguide">You must provide the following data attributes if you want the JS component to work</p>
-  <ul class="styleguide">
-    <li><code class="styleguide">data-ft-navigationbar</code> on the <code class="styleguide">.navigationbar</code> element</li>
-    <li><code class="styleguide">data-ft-navigationbar-toggle</code> on the <code class="styleguide">.navigationbar__toggle</code> element</li>
-  </ul>
-</div>
+### Data Attributes
+
+* `data-ft-navigationbar` on the `.navigationbar` element
+* `data-ft-navigationbar-toggle` n the `.navigationbar__toggle` element
+
+### Events
+
+Event                     | Description
+--------------------------| -----------------------
+`show.ft.navigationbar`   | The navigationbar is going to show
```

Comment:
Yes - you should still be able to listen in when the collapsed navbar is shown. If we modify that behavior, we should probably do the documentation changes in that PR.